### PR TITLE
[Doc] Fix `<RichTextInput>` import in SimpleForm doc

### PR DESCRIPTION
**Pull request by anthonycmain**
Minor correction to SimpleForm page to update RichTextInput reference

### CODE REVIEW

File: docs/SimpleForm.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ The `<SimpleForm>` creates a `<form>` to edit a record, and renders its children
 `<SimpleForm>` is often used as child of `<Create>` or `<Edit>`. It accepts Input and Field components as children. It relies on [react-hook-form](https://react-hook-form.com/) for form handling. It requires no prop by default.
 
 ```jsx
-import { Create, SimpleForm, TextInput, RichTextInput, NumberInput } from 'react-admin';
+import { Create, SimpleForm, TextInput, NumberInput } from 'react-admin';
+import RichTextInput from 'ra-input-rich-text';
```
Comment (adguernier) [inline docs/SimpleForm.md:20]:
```suggestion
import { RichTextInput } from 'ra-input-rich-text';
```

**Review by adguernier — CHANGES_REQUESTED** [review]

File: docs/SimpleForm.md (hunk 1 above)
Comment (adguernier) [inline docs/SimpleForm.md:20]:
```suggestion
import { RichTextInput } from 'ra-input-rich-text';
```

**Review by adguernier — CHANGES_REQUESTED** [review]

**Comment by adguernier** [comment]
Thanks for your contribution :+1:

**Review by slax57 — APPROVED** [review]
Thanks!

**Pull request merged**